File: java/cmf-eventing-api/src/main/java/cmf/eventing/patterns/rpc/IRpcEventBus.java
package cmf.eventing.patterns.rpc;

import cmf.bus.IDisposable;
import cmf.eventing.IEventBus;

/**
 * A convenience interface that combines the {@link IEventBus }, {@link IRpcSender} 
 * and {@link IRpcReceiver} interfaces together because they are most typically
 * implemented and referenced together when performing RPC type eventing.
 */
public interface IRpcEventBus extends IEventBus, IRpcSender, IRpcReceiver, IDisposable {

}

File: dotnet/source/cmf.eventing/patterns/rpc/IRpcEventBus.cs
using System;

namespace cmf.eventing.patterns.rpc
{
    /// <summary>
    /// A convenience interface that combines the <see cref="IEventBus">IEventBus</see>, 
    /// <see cref="IRpcRequestor">IRpcRequestor</see> and <see cref="IRpcResponder">IRpcResponder</see> 
    /// interfaces together because they are most typically implemented 
    /// and referenced together when performing RPC type eventing.
    /// </summary>
    public interface IRpcEventBus : IEventBus, IRpcRequestor, IRpcResponder
    {
    }
}
